.storybook/main.ts

```diff
@@ -0,0 +1,27 @@
+import type { StorybookConfig } from '@storybook/react-webpack5';
```

Comment:
**note:** with this setup we are going against what we want ( following the general practices ) https://nx.dev/recipes/storybook/configuring-storybook 

> Nx does not generate a root Storybook directory and shared root Storybook configurations. Instead, it only generates a Storybook configuration for each project in your workspace. You may still manually create a root Storybook configuration file, if it is needed for your use case.


I know this is how we use it in core but that's not something we should do long term.

Anyways for consistency reasons we can keep this as is, and when we will start the migration in core we can then fast forward to contrib.

**changes needed:**

in order to properly trigger affected pipelines we need to manually adjust project graph ( another "magic" disadvantage of this setup ) 
see "inputs" https://github.com/microsoft/fluentui/blob/master/nx.json#L23


**the optimal solution we wont long term:**
- project level specific configs
- anything share-able lives in standard monorepo package which is used in scoped per project configs - proper API contract

Reply:
Thank you for providing the background; it definitely clarifies things. We could potentially develop a local package with a storybook preset and integrate it across all contrib packages. However, I'm uncertain if it's beneficial since we'll eventually need to switch to the core version once it's available. In the meantime, as you recommended, I've added the "input" for the build-storybook  https://github.com/microsoft/fluentui-contrib/pull/244/files#diff-15552e50b1b7c05b05b7ffe08ee47b5ed62b8d2039229c58972a42fd22a7381cR25